torch_xla/csrc/ops/symeig.cpp

```diff
@@ -45,7 +45,7 @@ xla::Shape NodeOutputShape(const torch::lazy::Value& input, bool eigenvectors,
 }  // namespace
 
 SymEig::SymEig(const torch::lazy::Value& input, bool eigenvectors, bool lower)
-    : XlaNode(torch::lazy::OpKind(at::aten::symeig), {input},
+    : XlaNode(torch::lazy::OpKind(at::aten::linalg_eigh), {input},
```

Comment:
Should we remove symeig.h/.cpp since the operator is deprecated. Or you want to rename it to linalg_eigh in the 2nd PR which adds linalg_eigh?

Reply:
This file and implementation will be used for the new `linalg_eigh`.